selenium/che-selenium-test/src/main/java/org/eclipse/che/selenium/pageobject/ProjectExplorer.java

```diff
@@ -670,20 +674,25 @@ public void waitRemoveItemsByPath(String pathToItem) {
    * @param path item's path in format: "Test/src/pom.xml".
    */
   public void openContextMenuByPathSelectedItem(String path) {
+    final String itemXpath = getItemXpathByPath(path);
+
     for (int i = 1; ; i++) {
       waitAndSelectItem(path);
-
-      actionsFactory.createAction(seleniumWebDriver).contextClick().perform();
+      waitItemIsSelected(path);
+      seleniumWebDriverHelper.waitAndContextClick(By.xpath(itemXpath));
       waitContextMenu();
-
       try {
         waitItemIsSelected(path, REDRAW_UI_ELEMENTS_TIMEOUT_SEC);
```

Comment:
The command `waitItemIsSelected()` has been added above. 
Does it still make sense to check on item selection again after we obtained context menu?

Reply:
This block of code checks an error which described in the error message